java-checks/src/main/java/org/sonar/java/checks/SwitchLastCaseIsDefaultCheck.java

```diff
@@ -37,7 +37,7 @@
 public class SwitchLastCaseIsDefaultCheck extends IssuableSubscriptionVisitor {
 
   private static final String DEFAULT_LABEL_STRING = JavaKeyword.DEFAULT.getValue();
-  
+
```

Comment:
This change is not needed. Please revert it.
Let's try to keep the PR as small as possible. Minimal change, minimal impact and chances of side-effects, minimal effort for reviews, maximal efficiency.